fuzz_test/run_direct_test.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# used for manually running tests
+
+echo "=== Direct Noir Base64 Oracle Test ==="
+echo "This test compares Noir base64 implementation with Node.js base64"
+echo ""
+
+# Check if nargo is available
+if ! command -v nargo &> /dev/null; then
+    echo "Error: nargo command not found!"
+    echo "Please install nargo or add it to your PATH"
+    exit 1
+fi
+
+echo "Found nargo: $(which nargo)"
+
+# Check if Node.js is available
+if ! command -v node &> /dev/null; then
+    echo "Error: Node.js not found!"
+    echo "Please install Node.js to run the JavaScript RPC server"
+    exit 1
+fi
+
+echo "Found Node.js: $(which node)"
+
+# Install JavaScript dependencies if needed
+if [ ! -d "node_modules" ]; then
+    echo "Installing JavaScript dependencies..."
+    yarn install
+fi
```

Comment:
I think we can skip this. It forces us to be in `fuzz_test` when we invoke the script (otherwise the yarn install is relative to the wrong directory). It also requires us to do a `yarn install` on every test invocation.